libwallet/assets/dcr/dex_wallet.go

```diff
@@ -499,6 +545,40 @@ func (dw *DEXWallet) AddressPrivKey(ctx context.Context, addr stdaddr.Address) (
 }
 
 // Part of the Wallet interface.
-func (dw *DEXWallet) Reconfigure(_ context.Context, _ *dexasset.WalletConfig, _ dex.Network, _, _ string) (restart bool, err error) {
-	return false, nil
+func (dw *DEXWallet) Reconfigure(_ context.Context, _ *dexasset.WalletConfig, _ dex.Network, _ string) (restart bool, err error) {
+	return false, errors.New("Reconfigure not implemented by Cryptopower DEX wallet")
+}
+
+// These methods are part of Wallet interface but required only by the
+// dexasset.TicketBuyer interface, leave unimplemented.
+
+// PurchaseTickets purchases n tickets. vspHost and vspPubKey only
+// needed for internal wallets.
+func (dw *DEXWallet) PurchaseTickets(_ context.Context, _ int, _, _ string) ([]*dexasset.Ticket, error) {
+	return nil, errors.New("PurchaseTickets not implemented by Cryptopower DEX wallet")
+}
+
+// Tickets returns current active ticket hashes up until they are voted
+// or revoked. Includes unconfirmed tickets.
+func (dw *DEXWallet) Tickets(_ context.Context) ([]*dexasset.Ticket, error) {
+	return nil, errors.New("Tickets not implemented by Cryptopower DEX wallet")
+}
+
+// VotingPreferences returns current voting preferences.
+func (dw *DEXWallet) VotingPreferences(_ context.Context) ([]*walletjson.VoteChoice, []*dexasset.TBTreasurySpend, []*walletjson.TreasuryPolicyResult, error) {
+	return []*walletjson.VoteChoice{}, []*dexasset.TBTreasurySpend{}, []*walletjson.TreasuryPolicyResult{}, errors.New("VotingPreferences not implemented by Cryptopower DEX wallet")
+}
+
+// SetVotingPreferences sets preferences used when a ticket is chosen to
+// be voted on.
+func (dw *DEXWallet) SetVotingPreferences(_ context.Context, _, _, _ map[string]string) error {
+	return errors.New("SetVotingPreferences not implemented by Cryptopower DEX wallet")
+}
+
+func (dw *DEXWallet) SetTxFee(_ context.Context, _ dcrutil.Amount) error {
+	return errors.New("SetTxFee not implemented by Cryptopower DEX wallet")
+}
+
+func (dw *DEXWallet) StakeInfo(_ context.Context) (*dcrwallet.StakeInfoData, error) {
+	return nil, errors.New("StakeInfo not implemented by Cryptopower DEX wallet")
```

Comment:
> but I think you can just leave these off 

Do you mean to not return an error?

> since the interface is not satisfied it will error further up if core tries to use them.

Yes, this was what I had in mind. That way we can tell which methods are critical to core's functionality.

Reply:
> Do you mean to not return an error?

I think it returns a more generic error, like "this wallet is does not implement staking" It's fine to have custom errors. Was just commenting that I don't think it is strictly necessary to implement these. I could be wrong though.

Reply:
> Was just commenting that I don't think it is strictly necessary to implement these. I could be wrong though.

These methods were added to the dexdcr.Wallet interface (on upstream master) so we have to implement them.